### PR TITLE
Update starters.yml

### PR DESCRIPTION
**Pull request by markoradak**
Add the following Storybook starter: https://github.com/markoradak/gatsby-starter-storybook

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -1124,3 +1124,16 @@
     - Modular content elements relationships
     - Modular content relationships in rich text
     - Reverse link relationships
+- url: https://gatsby-starter-storybook.netlify.com/
+  repo: https://github.com/markoradak/gatsby-starter-storybook
+  description: Gatsby starter site with Storybook
+  tags:
+    - Storybook
+    - Styled Components
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
    - Styling:CSS-in-JS
```

File: docs/starters.yml [hunk 2]
```diff
@@ -1124,3 +1124,16 @@
...
+    - Netlify
```
Comment (LekoArts) [inline docs/starters.yml]: `Netlify` is not needed as I think it's too general

File: docs/starters.yml [hunk 3]
```diff
@@ -1124,3 +1124,16 @@
...
+    - ESLint
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
    - Linting
```

**Comment by markoradak** [comment]
Updated as per your feedback @LekoArts.
Sorry for misusing the ones I chosen initially.

**Pull request merged**